base/applications/mspaint/mouse.cpp

```diff
@@ -11,6 +11,10 @@
 
 #include "precomp.h"
 
+POINT pointStack[256];
+INT pointSP = 0;
+BOOL drawing = FALSE;
```

Comment:
Why do we need these globals here? Could they be statics of some class?

Reply:
Moved to `ToolBase`.